content/integrations/awsopsworks.md

```diff
@@ -4,6 +4,7 @@ integration_title: AWS OpsWorks
 kind: integration
 git_integration_title: amazon_ops_works
 newhlevel: true
+description: "Track AWS OpsWorks resource usage."
```

Comment:
Dont think this integration tracks resource usage. I think it tracks successes/failures and relevant run time metrics.

Reply:
identical to corporate website